### PR TITLE
Added vifm!

### PR DESCRIPTION
**Pull request by yigitemres**
Added vifm. (vi like file manager)

### CODE REVIEW

File: vifm.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "homepage": "https://vifm.info//",
```
Comment (r15ch13) [inline vifm.json]: Just a minor typo

**Comment by r15ch13** [comment]
You could have force pushed a clean version to the last pr. 😄

**Comment by yigitemres** [comment]
@r15ch13 Thx for typo warning. I somehow create a new branch inside of qutebrowser branch. (I'm sure that I was in master.) When I see pushes from another branch I simply do not want to push like that. 😕

**Comment by r15ch13** [comment]
I mean: copy the file > checkout master > delete messed up branch > create branch with the same name as before > add file > commit > force push
This way you could have clean up the other PR without creating a new one. 😄

**Comment by yigitemres** [comment]
I didn't know that. Thx for the tip.

**Pull request merged**